cypress/page/iframe.page.ts

```diff
@@ -0,0 +1,28 @@
+class IframePage {
+    private ifrmaPageURL: string;
+    private iframe: string;
+    private titleText: string;
+    private cssSelector: string;
+
+    constructor() {
+        this.ifrmaPageURL = "https://www.w3schools.com/html/html_iframe.asp";
+        this.iframe = "[title=\"W3Schools HTML Tutorial\"]";
+        this.titleText = "HTML Tutorial";
+        this.cssSelector = "[title=\"CSS Tutorial\"]";
+    }
+
+    public visit(): void {
+        cy.visit(this.ifrmaPageURL)
+        cy.wait(10000)
+    }
+
+    public getFrameTitle(): void {
```

Comment:
As the method says, you are getting the title, the assertion must be done in the cy.ts in this case